### DISCUSSION TITLE
Refer to parent container

### DISCUSSION THREAD

**Discussion by pleshevskiy** [terrastruct/d2#382] [Ideas]
I often find that I have to refer to the parent containers. And I have to rewrite the entire container and take all connections out of the container. I think it would be convenient to be able to refer to the parent container.

What about `..` ?

```d2
a: {
  e -> ..b.c
}

b: {
  c
}
```

**Comment by alixander** [comment]
oh this is a thing, i've been meaning to document it. it's underscore.

```
b
a: {
  _.b: "new label for b"
}
```

**Reply by pleshevskiy** [reply]
Great! Thanks!
